src/routes/Home.js

```diff
@@ -69,24 +72,32 @@ class Home extends Component {
         ];
 
         return (
-            <React.Fragment>
-                <PageHeader>
-                    <PageHeaderTitle title='Remediations'/>
-                    <TestButtons onRemediationCreated={ this.onRemediationCreated } />
-                </PageHeader>
-                <Main>
-                    <ConnectedRemediationTable loadRemediations={ this.loadRemediations } />
-                </Main>
-
-                <Wizard
-                    isLarge
-                    title="Create Plan"
-                    className='ins-c-plan-modal'
-                    onClose = { this.onClose }
-                    isOpen= { isModalOpen }
-                    content = { ModalStepContent }
-                />
-            </React.Fragment>
+            <PermissionContext.Consumer>
+                { value =>
+                    value.permissions.read === false &&
```

Comment:
This should only depend on the read permission - i.e. if the user does not have read permission they won't see anything (as the API won't return data without the read permission)